File: Java/src/main/java/com/google/devtools/poxoserializer/POxOSerializerClassPair.java
package com.google.devtools.poxoserializer;

import com.google.devtools.poxoserializer.serializers.GenericClassSerializer;

public class POxOSerializerClassPair {
  private Class<?> genericClass = null;
  
  private GenericClassSerializer serializer = null;

  public GenericClassSerializer getSerializer() {
    return serializer;
  }

  public void setSerializer(GenericClassSerializer serializer) {
    this.serializer = serializer;
  }

  public Class<?> getGenericClass() {
    return genericClass;
  }

  public void setGenericClass(Class<?> genericClass) {
    this.genericClass = genericClass;
  }
}

File: C#/POxO/POxOSerializerClassPair.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace POxO
{
    public class POxOSerializerClassPair
    {
        private Type genericClass = null;

        private GenericClassSerializer serializer = null;

        public GenericClassSerializer getSerializer()
        {
            return serializer;
        }

        public void setSerializer(GenericClassSerializer serializer)
        {
            this.serializer = serializer;
        }

        public Type getGenericClass()
        {
            return genericClass;
        }

        public void setGenericClass(Type genericClass)
        {
            this.genericClass = genericClass;
        }
    }
}
